tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -1749,7 +1749,7 @@ def _gpu_resource_check(slurm_commands, partition, instance_type, instance_type_
 def _test_slurm_version(remote_command_executor):
```

Comment:
Could we remove this slurm version check from the CLI and keep it just in the kitchen tests?
This will avoid us to create a patch for the CLI everytime we're updating slum version.

Reply:
And I'll go even further and remove also this check 

```
# Since we're installing PMIx v4.2.6, we expect to see pmix and pmix_v4 in the output.
    ...
    assert_that(mpi_list_output).matches(r"\s+pmix_v4($|\s+)")
```

Reply:
We may still need the patch for the Changelog.